crypto/x509/x509_vfy.c

```diff
@@ -3167,11 +3188,8 @@ static int build_chain(X509_STORE_CTX *ctx)
                         dane->pdpth = -1;
                 }
 
-                /*
-                 * Self-signed untrusted certificates get replaced by their
-                 * trusted matching issuer.  Otherwise, grow the chain.
-                 */
-                if (!self_signed) {
+                if (!self_signed) { /* untrusted not self-signed certificate */
```

Comment:
I see that there are preexisting instances of this comment style (on the same line as the opening brace of the block) in the file (e.g., line 3192 in the "left" side), but I don't see it covered in https://www.openssl.org/policies/technical/coding-style.html#chapter-8-commenting and it's unclear that we want to be adding more of them.

Reply:
The coding style guide says:
> Place comments above or to the right of the code they refer to.

so I see this covered.

Here I moved the part of the comment that refers to the condition to the right of the condition because this makes more clear what it refers to.